Gitter/Gitter/Gitter.WindowsPhone/Views/AboutPage.xaml

```diff
@@ -0,0 +1,81 @@
+            <Hub Header="About">
+                <HubSection Header="Information">
+                    <DataTemplate>
+                        <ScrollViewer>
+                            <Grid>
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto"/>
+                                    <RowDefinition Height="Auto"/>
+                                    <RowDefinition Height="Auto"/>
+                                    <RowDefinition Height="*"/>
+                                    <RowDefinition Height="Auto"/>
+                                </Grid.RowDefinitions>
+
+                                <StackPanel Margin="0 0 0 24">
+                                    <TextBlock Text="{Binding ApplicationTitle}"
+                                           Style="{ThemeResource SubheaderTextBlockStyle}"/>
+                                    <TextBlock Text="{Binding ApplicationVersion}"
+                                           Style="{ThemeResource TitleTextBlockStyle}"/>
+                                </StackPanel>
+
+                                <TextBlock Grid.Row="1"
+                                       Text="{Binding ApplicationDescription}"
+                                       Style="{ThemeResource TitleTextBlockStyle}"
+                                       TextWrapping="Wrap" />
+                                <HyperlinkButton Grid.Row="2"
+                                             Content="Having Problems? Click to raise an issue"
+                                             Margin="0,12,0,0"
+                                             HorizontalAlignment="Left"
+                                             FontSize="17.333"
+                                             FontWeight="Bold"
+                                             FontFamily="Segoe WP"
+                                             NavigateUri="https://github.com/Odonno/Modern-Gitter/issues/new"/>
+                                <StackPanel Grid.Row="4" Margin="12,0,12,30">
+                                    <Image Source="../Assets/Gitter.white.png"
+                                       Height="100"
+                                       HorizontalAlignment="Left"/>
+                                    <TextBlock Text="{Binding ApplicationPublisher}"
+                                           Style="{ThemeResource TitleTextBlockStyle}"/>
+                                </StackPanel>
+                            </Grid>
+                        </ScrollViewer>
+                    </DataTemplate>
+                </HubSection>
+                <HubSection Header="History">
+                    <DataTemplate>
+                        <ScrollViewer>
+                            <StackPanel>
+                                <TextBlock Style="{ThemeResource SubheaderTextBlockStyle}">Version 1.2.7</TextBlock>
```

Comment:
A better history log : 

* #0.3 (soon)
    * fix bug on realtime messages
    * fix bug on toast notification
    * parse effectively all the HTML sent (title, links, image, quote, formatted code)
    * search rooms by name
    * mention (toast) notifications
    * about page
* #0.2
    * copy/update/remove message (holding on a message)
    * toast notifications (unread messages)
    * toast notifications (realtime messages)
    * in-app notification as badge (unread count)
    * 'respond to' a chat user (tap on a chat avatar)
    * microphone enabled
* #0.1
    * main page (v1)
    * room page (v1)
    * push a message
    * add a link to this chat room (chat with us)